### ISSUE TITLE
[FR]: Immediate backup after power-on when octoprint is down on a given backup time 

### ISSUE THREAD

**Issue by coolduke666** [jneilliii/OctoPrint-BackupScheduler#10]
It's like cron vs anacron in linux environment. when you specifiy a backup time and your octoprint instance is down, the backup should start automatically once octoprint is up again and therfore a backup is done. 

I am printing only a few things per week, which means the I can't specify a date for backup because I don't know upfront, when my octoprint is running.

hope that makes sense :)

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by jneilliii** [comment]
Somewhat of a duplicate of #1. I like your idea better though of a "missed backup" running on startup rather than just every startup. Will keep that in mind for a future update to that feature request.
